account_claims.go

```diff
@@ -208,3 +208,8 @@ func (a *AccountClaims) ClearUserRevocation(pubKey string) {
 func (a *AccountClaims) IsUserRevoked(pubKey string, timestamp time.Time) bool {
 	return a.UserRevocations.IsRevoked(pubKey, timestamp)
 }
+
+// IsUserRevokedNow checks if the public key is in the revoked list with time.Now()
+func (a *AccountClaims) IsUserRevokedNow(pubKey string) bool {
```

Comment:
I feel this should just be IsUserRevoked(), and there could be on that IsUserRevokedAtTime() or something like that.